app/assets/stylesheets/stream_element.css.scss

```diff
@@ -12,7 +9,42 @@
   }
 }
 
+#main_stream > div > .photo {
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `main_stream` should be written in lowercase with hyphens